sdk/resourcemanager/azure-resourcemanager/pom.xml

```diff
@@ -137,7 +137,7 @@
     <dependency>
       <groupId>com.azure.resourcemanager</groupId>
       <artifactId>azure-resourcemanager-cosmos</artifactId>
-      <version>2.54.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure.resourcemanager:azure-resourcemanager-cosmos;dependency} -->
+      <version>2.53.4</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-cosmos;dependency} -->
```

Comment:
CI is complaining about this change. You can revert this change(as well as the `unreleased` entry in version_client.txt) and change it back to stable after release.

Reply:
Second thought, we may not be able to do this. Once it's merged to main, it'll break following FromSourceRun CIs.
Please keep the version as is. We may need to fix the sample. e.g. https://github.com/Azure/azure-sdk-for-java/pull/47006/files#diff-8be2e49ef8bfe60474277b05710d3a7c0794f859dbe3738cc6327d73f03a23a0